.mailmap

```diff
@@ -0,0 +1,6 @@
+Jose Cardona <cardona.josem@gmail.com>
+Jose Cardona <cardona.josem@gmail.com> <cardona.josem@gmail.com>
+Jose Cardona <cardona.josem@gmail.com> <jmcardon@my.yorku.ca>
+Jose Cardona <cardona.josem@gmail.com> <jose@debbie-ann.kek>
```

Comment:
LOL. Delet this.

Reply:
dude I almost made that your primary....

Reply:
:sob: